src/main/java/school/lemon/changerequest/java/generics/pair/PairUtils.java

```diff
@@ -0,0 +1,45 @@
+package school.lemon.changerequest.java.generics.pair;
+
+public class PairUtils {
+
+    public static boolean equals(Pair<?, ?> first, Pair<?, ?> second) {
+        return first.getKey().equals(second.getKey()) && first.getValue().equals(second.getValue());
+    }
+
+    public static <KeyType extends Comparable<KeyType>> int compareTo(Pair<KeyType, ?> first,
+                                                                      Pair<KeyType, ?> second) {
+        return first.getKey().compareTo(second.getKey());
+    }
+
+    public static Object[] getValues(Pair<?, ?>[] array) {
```

Comment:
You are working with generics.

I do want a return type to be specified.

Null-check?